docs/Guides/Prototype-Poisoning.md

```diff
@@ -7,20 +7,14 @@
 ## A Tale of (prototype) Poisoning
 <a id="pp"></a>
 
-This story is a behind-the-scenes look at the process and drama created by a
+Based on the article by Eran Hammer,the story is created by a
 particularity interesting web security issue. It is also a perfect illustration
 of the efforts required to maintain popular pieces of open source software and
 the limitations of existing communication channels.
 
-But first, if you use a JavaScript framework to process incoming JSON data, take
-a moment to read up on [Prototype
-Poisoning](https://medium.com/intrinsic/javascript-prototype-poisoning-vulnerabilities-in-the-wild-7bc15347c96)
-in general, and the specific [technical
-details](https://github.com/hapijs/hapi/issues/3916) of this issue. I'll explain
-it all in a bit, but since this could be a critical issue, you might want to
-verify your own code first. While this story is focused on a specific framework,
-any solution that uses `JSON.parse()` to process external data is potentially at
-risk.
+But first, if we use a JavaScript framework to process incoming JSON data, take
+a moment to read up on [Prototype Poisoning](https://medium.com/intrinsic/javascript-prototype-poisoning-vulnerabilities-in-the-wild-7bc15347c96)
+in general, and the specific [technical details](https://github.com/hapijs/hapi/issues/3916) of this issue. This could be a critical issue so, we might need to verify your own code first. While this story is focused on a specific framework however, any solution that uses `JSON.parse()` to process external data is potentially at risk.
```

Comment:
```suggestion
in general, and the specific [technical details](https://github.com/hapijs/hapi/issues/3916) of this issue.
This could be a critical issue so, we might need to verify your own code first.
While this story is focused on a specific framework however, any solution that uses `JSON.parse()` to process external data is potentially at risk.
```